### PR TITLE
Ds 448 stepbystep frontend development work

### PR DESCRIPTION
**Pull request by duncansanderson**
Adding a new "Styled List" component that has variations for Key Benefits (default), Step by step (Numbered), Content list (with image) and Key Statistics (Horizontal with image)

### CODE REVIEW

**Review by mjpvs — APPROVED** [review]

File: src/components/styled-list/StyledList.vue [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+<script setup>
+import VsHeading from '@/components/heading/Heading.vue';
+import {
+    VsContainer,
+    VsRow,
+    VsCol,
+} from '@/components/grid';
+
+const props = defineProps({
+    anchorLink: {
```
Comment (GemmaLouise) [inline src/components/styled-list/StyledList.vue:45]: All the props need comments

File: src/components/styled-list/components/StyledListItem.vue [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                    v-if="props.variant === 'image-horizontal'"
+                    :src="props.imageSrc"
+                />
+
+                <VsHeading level="3">
+                    {{ heading }}
+                </VsHeading>
+
+                <VsRichTextWrapper>
+                    <slot />
```
Comment (GemmaLouise) [inline src/components/styled-list/components/StyledListItem.vue:25]: Slots need comments too - take a look at https://visitscotland.atlassian.net/wiki/spaces/DE/pages/159088753/Vue

File: src/components/styled-list/StyledList.vue (hunk 1 above)
Comment (duncansanderson) [reply src/components/styled-list/StyledList.vue:45]: I've added comments to all props

File: src/components/styled-list/components/StyledListItem.vue (hunk 2 above)
Comment (duncansanderson) [reply src/components/styled-list/components/StyledListItem.vue:25]: I've added comments to all slots

**Review by GemmaLouise — APPROVED** [review]

**Pull request merged**